pennylane_orquestra/orquestra_device.py

```diff
@@ -382,3 +387,254 @@ def qubit_operator_string(self, observable):
         # Use consecutive integers as default wire_map
         wire_map = {v: idx for idx, v in enumerate(self.wires)}
         return _terms_to_qubit_operator_string(coeffs, obs_list, wires=wire_map)
+
+    def execute(self, circuit, **kwargs):
+        # Input checks
+        not_all_expval = any(obs.return_type is not Expectation for obs in circuit.observables)
+        if not_all_expval:
+            raise NotImplementedError(
+                f"The {self.short_name} device only supports returning expectation values."
+            )
```

Comment:
Good idea to check this. The other alternative (not defining the `var` and `sample` methods) would result in an error message only occurring _after_ the job returns from the server (if it gets that far).